compose/src/main/java/com/bottlerocketstudios/compose/pullrequest/PullRequestScreen.kt

```diff
@@ -0,0 +1,89 @@
+package com.bottlerocketstudios.compose.pullrequest
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.bottlerocketstudios.compose.resources.Dimens
+import com.bottlerocketstudios.compose.resources.brown_grey
+import com.bottlerocketstudios.compose.util.Preview
+import com.bottlerocketstudios.compose.util.asMutableState
+
+@Composable
+fun PullRequestScreen(state: PullRequestScreenState) {
+    Column() {
+        Divider(
+            modifier = Modifier.padding(start = 9.dp, end = 9.dp, top = 24.dp, bottom = 24.dp),
+            color = brown_grey,
+            thickness = 1.dp
+        )
+        if (state.pullRequestList.value.isNotEmpty()) {
+            LazyColumn(
+                verticalArrangement = Arrangement.spacedBy(Dimens.grid_1_5),
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                items(
+                    items = state.pullRequestList.value,
+                    itemContent = { item -> PullRequestItemCard(item) }
+                )
+            }
+        } else {
+            PullRequestEmptyScreen()
+        }
+        Divider(
+            modifier = Modifier.padding(start = 9.dp, end = 9.dp, top = 24.dp, bottom = 24.dp),
+            color = brown_grey,
+            thickness = 1.dp
+        )
+    }
+}
+
+@Preview
+@Composable
+fun PullRequestsPreview() {
```

Comment:
@Br-DiorD nit: Make `private`

```suggestion
private fun PullRequestsPreview() {
```